.github/workflows/django.yml

```diff
@@ -0,0 +1,70 @@
+name: "Iconica Standard Django CI Action"
+
```

Comment:
i typically also version the workflow, so we have a version of the actual workflow and a version for the starter workflow (which only changes if extra inputs are added)